test/e2e.sh

```diff
@@ -0,0 +1,166 @@
+#! /bin/bash
+
+set -eux -o pipefail 
+
+# Gets the /etc/os-release file from a worker node host OS
+# Sends the resulting filename to stdout
+get_node_os_release_file() {
+    os_release_file=${ARTIFACT_DIR}/etc-os-release
+    if [[ ! -s "${os_release_file}" ]]; then 
+	    worker_node=$(kubectl get nodes --selector='node-role.kubernetes.io/worker' -ojsonpath='{.items[].metadata.name}')
+
+	    oc debug node/${worker_node} \
+            --quiet \
+            -- cat /host/etc/os-release > ${os_release_file}
+    fi
+
+    echo ${os_release_file}
+}
+
+
+# Get running RHCOS version. Assumes all nodes are running the same version.
+# TODO check RT kernel version
+get_node_rhcos_version() {
+    os_release_file=$(get_node_os_release_file)
+    rhcos_version=$(cat ${os_release_file} \
+	    | grep "OSTREE_VERSION" \
+            | cut -d= -f2 \
```

Comment:
Pipes are not aligned here, unlike in `get_node_rhel_version` below. Could you please remove the extra whitespaces?

Reply:
+1 for cutting the `OSTREE_VERSION=...` line and storing the contents in `${rhcos_version}` explicitely 👍🏻

Reply:
Good catch, I had a mix of tabs and spaces. Should be fixed now.